### PR TITLE
[v2.0] Updates in Overview page and Address details modal

### LINKED ISSUE #567: Implement new Address page side modal UI

### LINKED ISSUE #568: Implement updated Overview page header with floating shortcuts

### PR DESCRIPTION
**Pull request by nop33**
Closes #567 

Closes #568

(updated images)

image

image

### CODE REVIEW

**Comment by nop33** [comment]
The position of the error is slightly different than in Figma so that it works well for both plain input fields, but also "combo" input fields (like the addresses inputs in the send modal). Lemme know if it should be changed!

image
image

**Comment by nop33** [comment]
Updated the last step of the send modal with a green check ✅ 


https://user-images.githubusercontent.com/1579899/224745811-0d4d6b6a-a726-4d79-95e6-9e1594935e9a.mp4

**Comment by mvaivre** [comment]
> I hope you don't mind that I kept adding this on this PR! It's relatively small and since you haven't reviewed it yet, I thought we could bundle some more things together :) 

I really don't mind because I'm used to do the same hehe :P

File: src/storage/app-state/slices/settingsSlice.ts [hunk 1]
```diff
@@ -59,7 +65,14 @@ const settingsSlice = createSlice({
     }
   },
   extraReducers(builder) {
-    builder.addCase(localStorageDataMigrated, () => SettingsStorage.load('general') as GeneralSettings)
+    builder
+      .addCase(localStorageDataMigrated, () => SettingsStorage.load('general') as GeneralSettings)
+      .addCase(systemLanguageMatchSucceeded, (state, action) => {
```
Comment (mvaivre) [inline src/storage/app-state/slices/settingsSlice.ts:70]: Nit: To make it easier to see what the payload is, we could do:

```ts
.addCase(systemLanguageMatchSucceeded, (state, { payload: language }) => {
  state.language = language
})
```

**Review by mvaivre — APPROVED** [review]
Very nice work 🤙 

Ok to merge after discussing the tiny couple of comments I sent 🚀

File: src/storage/app-state/slices/settingsSlice.ts (hunk 1 above)
Comment (nop33) [reply src/storage/app-state/slices/settingsSlice.ts:70]: Absolutely!

**Comment by mvaivre** [comment]
I'm in favour of arranging them in a row because it's how they are displayed (or gonna be displayed) in the address view and the transfers page :)

**Pull request merged**